kratos/python/add_matrix_to_python.cpp

```diff
@@ -89,6 +90,33 @@ namespace Python
 
           return matrix;
         }));
+
+        matrix_binder.def(py::init([](const py::list& input){
+          std::size_t num_rows = input.size();
+          std::size_t num_cols = 0;
+
+          //To calculate the number of columns
+          for(std::size_t i = 0 ; i< input.size() ; i++)
+          {
+            auto v_row = Vector(py::cast<Vector>(input[i]));
+            if(num_cols < v_row.size())
+              num_cols = v_row.size();
+          }
+
+          DenseMatrix<double>matrix = DenseMatrix<double>(num_rows, num_cols);
+          for(std::size_t i=0; i<num_rows; i++){
+            auto v_row = Vector(py::cast<Vector>(input[i]));
+            for(std::size_t j=0; j<num_cols; j++){
+              if(j < v_row.size())
+                matrix(i,j) = v_row[j];
+              else
+                matrix(i,j) = 0.0;
```

Comment:
I wouldn't open the possibility to set different size row data (I'd ask the user for those zeros).

Reply:
I agree, the input must have the correct size

check [here](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/sources/kratos_parameters.cpp#L603-L627), this does sth similar
I would do the same